share/completion/fzy

```diff
@@ -0,0 +1,29 @@
+# (C) 2024 unrealapex
+
+# Completion script for fzy
+
+function completion/fzy {
+
+        typeset OPTIONS ARGOPT PREFIX
+        OPTIONS=( #>#
+        "l --lines:; how many lines to display"
+        "p --prompt:; input prompt"
+        "s --show-scores:; show the scores for each item"
+        "t --tty:; use tty instead of the default tty device"
+        "q --query:; use query as the initial search query"
+        "e --show-matches:; non-interactive mode"
```

Comment:
I don't know much about fzy, but if a long option requires an argument, I assume the short option does too, so we should have a colon flag for the short option as well.

According to the [manpage](https://github.com/jhawthorn/fzy/blob/5ae3b953318becfafc1a54b994fce4ec7a8d99a8/fzy.1), the `--show-scores` option does not seem to take an argument.

```suggestion
        "l: --lines:; how many lines to display"
        "p: --prompt:; input prompt"
        "s --show-scores; show the scores for each item"
        "t: --tty:; use tty instead of the default tty device"
        "q: --query:; use query as the initial search query"
        "e: --show-matches:; non-interactive mode"
```